src/CalcViewModel/Common/Utils.cpp

```diff
@@ -86,11 +86,11 @@ bool Utils::IsLastCharacterTarget(_In_ wstring const &input, _In_ wchar_t target
     return !input.empty() && input.back() == target;
 }
 
-//return wstring after removing characters like space, comma, and double quotes
+//return wstring after removing characters like space, comma, double quotes, and monetary prefix currency symbols supported by the Windows keyboard
 wstring Utils::RemoveUnwantedCharsFromWstring(wstring input)
 {
-    wchar_t unWantedChars[] = { L' ', L',', L'"', 8234, 8235, 8236, 8237 };
-    return RemoveUnwantedCharsFromWstring(input, unWantedChars, 6);
+    wchar_t unWantedChars[] = { L' ', L',', L'"', 165, 164, 8373, 36, 8353, 8361, 8362, 8358, 8377, 163, 8364, 8234, 8235, 8236, 8237 };
```

Comment:
Let's also modernize this and use std::array<wchar_t>.

Reply:
Why would this be better as opposed to keep the current conventions?

What are the benefits of changing it?

Reply:
std::array provides you with the size, so you don't have to worry about calculating the length correctly. static constexpr causes the value to be calculated at compile-time vs runtime.

Reply:
I've also read a little and it does feels better. It's been a while since I've used C++ though. Would we still be able to pass `std::array<wchar_t>' as a parameter where the function expects a `wchar_t[]`

Reply:
You should still be able to get a pointer to the first element. However if that doesn't work we should just update the function so it does understand std::array. Ideally we would use something like ahn array_view, but that's not part of the standard yet.

Reply:
@mikaelmello, is there a specific example function you're thinking of?  Functions don't typically accept primitive arrays as the parameter type, ie, `wchar_t[]`.  When declared that way, the type of the variable [decays](https://stackoverflow.com/questions/29346552/passing-an-array-to-a-function-formal-parameter) to `wchar_t*`.

Instead of passing the `std::array` to another function, the best approach would be to pass begin and end iterators over a sequence.  Then the function can accept input from other types, such as `vector`

Reply:
@jlaanstra, I see, thanks for the explanation!

@danbelcher-MSFT, no, you're right about the function expecting a * (and not a []). I just thought that maybe the way that `std::array` was built could be somewhat compatible? Apparently not. I like the approach you said, specially the "Then the function can accept input from other types, such as vector" part. Thank you also for taking your time and explaining!